app/routes/docs.faq.md

```diff
@@ -66,6 +66,30 @@ This warning means that there have not been any recent alerts on that topic.
 
 To get started, [sign in or sign up](https://gcn.nasa.gov/login) and then select 'Email Notifications' from account dropdown menu. See also [GCN Circular 32517](https://gcn.gsfc.nasa.gov/gcn3/32517.gcn3).
 
+## Notice Producers
+
+### Which unit system is recommended to be used?
+
+To ensure a consistent database for same property, but different instrument, it is preferable to utilize SI units or maintain the same unit system. Easy conversion can be made using the astropy software.
+
+### Which option, "allOf", "anyOf", or "oneOf" should be used?
+
+We recommend utilizing the "allOf" option, as none of the properties in the core schema are required. However, this flexibility, with no required property, can lead to false positives when "anyOf" is used, as anything will technically validate against it.
+
+### How can I refer the coincident detection with another instrument?
```

Comment:
This sentence seems non-grammatical. @jracusin, would you please help re-word it?